submissions/the-darkesttt/js-post-oop/index.js

```diff
@@ -0,0 +1,71 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/the-darkesttt/a-tiny-JS-world
+   Web app: https://the-darkesttt.github.io/a-tiny-JS-world/
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Creature {
+   species;
+   name;
+   gender;
+   saying;
+   legs;
+   hands;
+
+   constructor( name, gender, saying) {
+      this.name = name;
+      this.gender = gender;
+      this.saying = saying;
+   }
+
+   showValues(obj) {
+      print(Object.values(obj).join('; '));
```

Comment:
`Object` methods like `keys`, `values`, `entries` shouldn't be used when a particular order is required as these do not guarantee any particular order of keys/values. Same refers to `for...of` and `for...in` when applied to objects.